.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
is there any reason to enforce this specific rules on .md files ?

Reply:
Yeah. Because 2 trailinf spaces are used for breaking a line in Markdown.